.github/workflows/test_install.yml

```diff
@@ -0,0 +1,20 @@
+permissions: read-all
```

Comment:
Can you follow the file naming convention we use on the generator https://github.com/slsa-framework/slsa-github-generator/tree/main/.github/workflows?
Basically have pre-submit.actions.yml for Action presubmit, etc.

We only had a pre-submit.yml in this repo, but looks like it's time to change.